addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -200,18 +199,20 @@ patch(Chatter.prototype, {
                 const text = partner.email ? partner.emailWithoutDomain : partner.name;
                 return `<span class="text-muted" title="${escape(
                     partner.email || _t("no email address")
-                )}">${escape(text)}</span>`;
+                )}"> ${escape(text)}</span>`;
             });
+        return markup(recipients);
+    },
+
+    get recipientsCount() {
         if (this.state.thread && this.state.thread.recipients.length > 5) {
-            recipients.push(
-                escape(
-                    _t("%(recipientCount)s more", {
-                        recipientCount: this.state.thread.recipients.length - 5,
-                    })
-                )
-            );
+             return escape(
+                 _t("out of %(recipientCount)s", {
```

Comment:
It's not acceptable to split translation phrases in several parts as order might need to be changed in different languages.

There should be different complete translation phrases for the different possible cases.